charts/karpor/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: karpor
-version: 0.7.5
+version: 0.7.6
 type: application
```

Comment:
The version bump from 0.7.5 to 0.7.6 follows semantic versioning correctly for a minor update.